SwiftI18n/Classes/Main/I18nManager.swift

```diff
@@ -12,13 +12,27 @@ public class I18nManager {
 
     public static var instance = I18nManager()
     
+    public var overrideToFallbackLanguage = false
```

Comment:
As agreed in the comments, feel free to remove this one and just keep `fallbackLanguage: String?`.